src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -94,4 +94,8 @@ object IndexConstants {
   val OPTIMIZE_MODE_QUICK = "quick"
   val OPTIMIZE_MODE_FULL = "full"
   val OPTIMIZE_MODES = Seq(OPTIMIZE_MODE_QUICK, OPTIMIZE_MODE_FULL)
+
+  val UNKNOWN_FILE_ID: Long = -1L
+  private[hyperspace] val MAX_FILE_ID = "maxFileId"
+  private[hyperspace] val DEFAULT_MAX_FILE_ID = UNKNOWN_FILE_ID.toString
```

Comment:
Any reason we are using both UNKNOWN_FILE_ID and DEFAULT_MAX_FILE_ID for the same value?

Reply:
No, there is no dependency in code on that. Just changed DEFAULT_MAX_FILE_ID  explicitly to "-1".